chore(common): add product page links (#983)
telerik/blazor-docs: components/barcodes/barcode/overview.md

# Blazor Barcode Overview

> The <a href="https://www.telerik.com/blazor-ui/barcode" target="_blank">Blazor Barcode component</a> represents data in a machine-readable format. You can choose from several supported standards.

Reviewer: ```suggestion
The <a href="https://www.telerik.com/blazor-ui/barcode" target="_blank">Blazor Barcode component</a> represents data in a machine-readable format. It supports several different encoding standards.
```
